### PR TITLE
UX/1785 cluster create wizard stuck in loading state after submit api request errored out

### PR DESCRIPTION
**Pull request by patil-pratik-87**
Screenshot 2022-04-13 at 8 22 03 PM
Screenshot 2022-04-13 at 9 46 23 PM

### CODE REVIEW

**Comment by patil-pratik-87** [comment]
Getting other wizards error handling done in another PR

File: src/app/core/components/wizard/WizardSummary.tsx [hunk 1]
```diff
@@ -66,7 +68,10 @@ export default function WizardSummary<T>({
           {title} {titleStepLabel}
         </Text>
       </header>
-      <section className={classes.wizardBody}>{children}</section>
+      <section className={classes.wizardBody}>
+        {!!error && <Alert variant="error" title={error?.title} message={error?.message} />}
```
Comment (jruddell) [inline src/app/core/components/wizard/WizardSummary.tsx:72]: maybe its more appropriate for this to show up in the right hand summary section instead

Comment (patil-pratik-87) [reply src/app/core/components/wizard/WizardSummary.tsx]: sure, did this 
Screenshot 2022-04-13 at 11 45 23 PM

**Comment by patil-pratik-87** [comment]
> Getting other wizards error handling done in another PR

Never mind. I am done so, put them in this PR itself

**Review by jruddell — APPROVED** [review]

File: src/app/core/components/wizard/WizardSummary.tsx (hunk 1 above)
Comment (jruddell) [reply src/app/core/components/wizard/WizardSummary.tsx]: 😍

**Pull request merged**